packages/blade/scripts/generateBundleSizeInfo.js

```diff
@@ -0,0 +1,115 @@
+const fs = require('fs');
+const path = require('path');
+const babelParser = require('@babel/parser');
+const traverse = require('@babel/traverse').default;
+const execa = require('execa');
+
+const main = async () => {
+  const { globby } = await import('globby');
+
+  const indexPaths = await globby([
+    'build/lib/web/production/components/**/index.js',
+    '!build/lib/web/production/components/{Icons,Form}/**/index.js',
+    '!build/lib/web/production/components/index.js',
+    '!build/lib/web/production/components/Box/styledProps/index.js',
+  ]);
+
+  const excludedComponents = [
+    'useTheme',
+    'announce',
+    'clearAnnouncer',
+    'destroyAnnouncer',
+    'getTextProps',
+    'screenReaderStyles',
+    'useActionListContext',
+    'ActionListItemAsset', // This is just an <img> tag
+    // These are same as Badge, Amount, Counter, Link, Text components
+    'CardHeaderAmount',
+    'CardHeaderBadge',
+    'CardHeaderCounter',
+    'CardHeaderLink',
+    'CardHeaderText',
+    'ComponentIds',
+    // Accessibility components
+    'SkipNavContent',
+    'SkipNavLink',
+    'VisuallyHidden',
+  ];
+  const sizes = [];
+
+  // Get all the components name exported from the bundle and add them to the size-limit configuration
+  indexPaths.forEach((indexPath) => {
+    const fileContent = fs.readFileSync(path.resolve(__dirname, `../${indexPath}`), 'utf8');
+
+    // Parse the file content to generate an AST
+    const ast = babelParser.parse(fileContent, {
+      sourceType: 'module',
+    });
+    // Arrays to store size-limit stats and component names to exclude
+    const exportedComponents = [];
+
+    // Traverse the AST to get the component names
+    traverse(ast, {
+      // Get the component name from the export statement
+      ExportSpecifier: ({ node }) => {
+        const componentName = node.exported.name;
```

Comment:
Our codebase already follows this convention, we have to provide named exports for our components because we allow all imports from `@razorpay/blade/components`.